lib/src/gitignore.rs

```diff
@@ -76,8 +76,6 @@ impl GitIgnoreFile {
                     line: String::from_utf8_lossy(input_line).to_string(),
                     source: err,
                 })?;
-            // FIXME: do we need to provide the `from` argument? Is it for providing
-            // diagnostics or correctness?
             builder
```

Comment:
nit: maybe we can leave a comment saying that "from argument doesn't provide any diagnostics or correctness"?